packages/connectivity_plus/connectivity_plus/ios/Classes/ReachabilityConnectivityProvider.swift

```diff
@@ -4,15 +4,18 @@ import Reachability
 public class ReachabilityConnectivityProvider: NSObject, ConnectivityProvider {
   private var _reachability: Reachability?
 
-  public var currentConnectivityType: ConnectivityType {
-    let reachability = ensureReachability()
```

Comment:
What is the reason here to change from using `ensureReachability()` to use the `_reachability` property?

Reply:
Returned it back, it my personal preferences, cause sometimes is confusing where you use guard construction in the code.